apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -1069,15 +1157,30 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
                      %{"amount" => 2, "currency" => @other_token_hex, "owner" => bob_addr},
                      %{"amount" => 3, "currency" => @other_token_hex, "owner" => bob_addr}
                    ],
-                   "fee" => %{"amount" => 5, "currency" => @eth_hex}
+                   "fee" => %{"currency" => @default_fee_currency}
                  }
                )
     end
 
-    @tag fixtures: [:alice, :more_utxos]
+    @tag fixtures: [:alice, :more_utxos, :test_server]
     test "transaction without payments that burns funds in fees is created correctly and incorrect on decoding",
-         %{alice: alice} do
-      fee = 15
+         %{alice: alice, test_server: context} do
+      fee_response = %{
+        @str_tx_type => [
+          %{
+            "currency" => @other_token_hex,
+            "amount" => 15,
```

Comment:
here, I don't know where the `15` used to come from, it seems strange - have you tried with the default? If it works with the default, you can simplify here